recipe/meta.yaml

```diff
@@ -118,6 +159,30 @@ outputs:
       license: BSD-3-Clause
       license_file: LICENSE
 
+  - name: gfortran
+    script: install-symlinks.sh
+    build:
+      skip: True  # [target_platform != ctng_target_platform]
+    requirements:
+      - gfortran_{{ target_platform }} {{ ctng_gcc }}.*
```

Comment:
@isuruf , why does the gfortran symlink depend on the gfortran activation script here?  Did you mean to have `gfortran_impl_{{ target_platform }} {{ ctng_gcc }}.*`?

Reply:
`gfortran, gcc, gxx` packages are user-facing packages and shouldn't be used in conda-build. So it makes sense to use the activation scripts so that the *FLAGS env variables are exposed to the user.

Reply:
I understand that they are user-facing and shouldn't be used in conda-build.  

`gfortran` is depending differently than `gcc` and `gxx`.  I think that we should make them consistent.

I think there is a  great usecase for having the symlinks **not** depend on the activation scripts. When you have a cross-compiler installed in your environment, the activation script will set FLAGS for the HOST.  Having the symlinks installed for BUILD would be helpful in some situations to have a working BUILD toolchain.  The conventions for passing FLAGS to the BUILD toolchain seem less clear than to HOST.  For example, CFLAGS_FOR_BUILD. I would propose that we make `gfortran` match what `gcc` and `gxx` are doing.

Reply:
Ah, I didn't realize that gfortran acts differently. Yes, let's fix gfortran.

Reply:
Opened #69 to fix.